core/trino-spi/src/main/java/io/trino/spi/block/MethodHandleUtil.java

```diff
@@ -24,6 +24,7 @@
 import static io.trino.spi.StandardErrorCode.GENERIC_INTERNAL_ERROR;
 import static java.lang.String.format;
 
+@Deprecated
```

Comment:
`MethodHandleUtil` is still used in the Trino code. What should be used instead?